modules/rostests/apitests/apphelp/CMakeLists.txt

```
@@ -1,5 +1,7 @@
 project(appcompat)
-add_definitions(-D__ROS_LONG64__ -DWINETEST_USE_DBGSTR_LONGLONG)
+add_definitions(-D__ROS_LONG64__ -DWINETEST_USE_DBGSTR_LONGLONG -DUNICODE -D_UNICODE)
+set_cpp(WITH_RUNTIME)
+
 
 include_directories(${CMAKE_CURRENT_BINARY_DIR})
```

If my guess is correct, this should fix it.
Does it look right?
```suggestion
include_directories(
    ${REACTOS_SOURCE_DIR}/sdk/lib/atl
    ${CMAKE_CURRENT_BINARY_DIR})
```